src/cmd/linuxkit/run_vcenter.go

```diff
@@ -90,7 +92,7 @@ func runVcenter(args []string) {
 	}
 
 	// Test any passed in files before creating a new VM
-	checkFile(*newVM.path)
+	// checkFile(*newVM.path)
```

Comment:
Could you either delete this or, if it is temporary, add a comment in the form of `// TODO(github user name): ....` explaining why this commented out